tests/components/teksavvy/test_sensor.py

```diff
@@ -126,43 +127,43 @@
     await async_setup_component(hass, "sensor", {"sensor": config})
 
     state = hass.states.get("sensor.teksavvy_data_limit")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "inf"
 
     state = hass.states.get("sensor.teksavvy_off_peak_download")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "36.24"
 
     state = hass.states.get("sensor.teksavvy_off_peak_upload")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "1.58"
 
     state = hass.states.get("sensor.teksavvy_off_peak_total")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "37.82"
 
     state = hass.states.get("sensor.teksavvy_on_peak_download")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "226.75"
 
     state = hass.states.get("sensor.teksavvy_on_peak_upload")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "8.82"
 
     state = hass.states.get("sensor.teksavvy_on_peak_total")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "235.57"
 
     state = hass.states.get("sensor.teksavvy_usage")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
     assert state.state == "226.75"
 
     state = hass.states.get("sensor.teksavvy_usage_ratio")
     assert state.attributes.get("unit_of_measurement") == "%"
     assert state.state == "0"
 
     state = hass.states.get("sensor.teksavvy_remaining")
-    assert state.attributes.get("unit_of_measurement") == "GB"
+    assert state.attributes.get("unit_of_measurement") == DATA_GIGABYTES
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5001170)